src/styles/contactGroupCreation.styl

```diff
@@ -15,16 +15,17 @@
     opacity 0
 
 /*since we keep the existing style of
-react-select for the menu and we only 
-want to override with & zindex... important 
+react-select for the menu and we only
+want to override with & zindex... important
 is important */
 .react-select__menu
     width rem(384) !important
     z-index 100000 !important
+    right 0
```

Comment:
Why do we have to add this now? If it's for a good reason, we should update the comment above as well.

Reply:
As the select goes to te right of the modal, we want the dropdown to start from the right to the left.
This was a design request. I'll update the comment.